apps/ucp-ui/src/shared/hooks/useSuccessSnackbar.ts

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+
+const useSuccessSnackbar = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [message, setMessage] = useState<string>();
+
+  const handleCloseSuccessSnackbar = () => setIsOpen(false);
+  const handleOpenSuccessSnackbar = () => setIsOpen(true);
+  const handleOpenSuccessSnackbarWithMessage = (newMessage: string) => {
+    setMessage(newMessage);
+    handleOpenSuccessSnackbar();
+  };
+
+  return {
+    handleOpenSuccessSnackbarWithMessage,
+    successSnackbarProps: {
+      message,
+      onClose: handleCloseSuccessSnackbar,
+      open: isOpen,
```

Comment:
`open` sounds like a function but it's a bool, why not use `isOpen`?

Reply:
open is in the <Snackbar /> api. I'm just spreading these props to a snackbar, so unless I write a wrapper around <Snackbar /> I have to use open

Reply:
Yeah, he's just following what the MUI Snackbar components needs.